automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -43,10 +43,23 @@ object MachineConfig {
   )
 }
 
+case class ServiceAccountInfo(clusterServiceAccount: Option[WorkbenchEmail],
+                              notebookServiceAccount: Option[WorkbenchEmail])
+
+object ServiceAccountInfo {
+  // TODO: something less hacky, please!
+  // If we're going to use Jackson we should use it the right way, with annotations in our model.
+  // Otherwise we should rip out LeonardoModelCopy + ClusterKluge and just use Leo model objects + spray json (my prefrence).
```

Comment:
+1 publish leo-model and use it in swat. both leo-swat and our jupyter-docker are both in need of some serious love once we're AoU-ready.

Reply:
Sounds good